src/code/z_scene.c

```diff
@@ -498,7 +498,7 @@ void (*gSceneCmdHandlers[SCENE_CMD_ID_MAX])(PlayState*, SceneCmd*) = {
     Scene_CommandMiscSettings,        // SCENE_CMD_ID_MISC_SETTINGS
 };
 
-RomFile sNaviMsgFiles[] = {
+RomFile sNaviHintFiles[] = {
     ROM_FILE(elf_message_field),
     ROM_FILE(elf_message_ydan),
     ROM_FILE_UNSET,
```

Comment:
As an aside, afaik there's no evidence `sNaviHintFiles[2]` exists, and the code doesn't plan for `ROM_FILE_UNSET` anyway

So I would think `sNaviHintFiles[2]` is actually an unk (or file padding?)

Reply:
file padding sounds right, object table stuff should prob be in its own. would be good to get some other version's approval

                0x0000000080127508                sNaviMsgsFiles
                0x0000000080127520                gLinkObjectIds
                0x0000000080127524                gObjectTableSize
                0x0000000080127528                gObjectTable
                
 MM also agrees

Reply:
we can investigate this further and make this change in another pr